<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oknotablitsa</class>
 <widget class="QDialog" name="oknotablitsa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1052</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="8">
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>сохранить параметры</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>запас размера пакета</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>длина эпохи,мс</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>загрузка каналов %</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="3" column="3">
     <layout class="QGridLayout" name="gridLayout_2"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>шаг между тайм-слотами</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="3" column="7">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>количество тайм-слотов</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </item>
    <item row="2" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="4" column="7">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="4" column="5">
     <widget class="QLineEdit" name="lineEdit">
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="1" column="0" colspan="9">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Рекомендуемая длина эпохи, мс:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1063</x>
     <y>400</y>
     <width>133</width>
     <height>132</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>закрасить тайм-слот</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>очистить тайм-слот</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>очистить таблицу</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>сохранить таблицу</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1064</x>
     <y>590</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>проверить</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>1064</x>
     <y>50</y>
     <width>121</width>
     <height>130</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>ID трафика</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>тайм-слот</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>задать задержку</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>oknotablitsa</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>hide(label)</slot>
 </slots>
</ui>
